pkg/agent/plugin/workloadattestor/docker/docker_posix_test.go

```diff
@@ -71,6 +72,20 @@ func TestContainerExtraction(t *testing.T) {
 			desc:    "default finder does not match cgroup missing docker prefix",
 			cgroups: "4:devices:/system.slice/41e4ab61d2860b0e1467de0da0a9c6068012761febec402dc04a5a94f32ea867.scope",
 		},
+		{
+			desc:    "default finder matches cgroups v2 with systemd driver",
+			cgroups: testCgroupV2Entries,
+			hasMatch: true,
+		},
+		{
+			desc:    "custom matchers match cgroups v2 with systemd driver",
+			cgroups: testCgroupV2Entries,
+			cfg: `container_id_cgroup_matchers = [
+"/system.slice/*/<id>", # systemd-managed cgroups
+"/docker/<id>", # Docker-managed cgroups
```

Comment:
is `"/docker/<id>"` needed? the input is only `0::/system.slice/system-unit.slice/docker-6469646e742065787065637420616e796f6e6520746f20726561642074686973.scope`

Reply:
The intent was to have multiple patterns in the test to have multiple patterns, but it is not strictly needed.